src/pages/TeamPage/index.js

```diff
@@ -241,7 +240,7 @@ function TeamPage() {
                         fontWeight: "bold",
                         borderBottom: "3px solid #48a145",
                       }
-                    : {}
+                    : { color: "#9c9999" }
```

Comment:
Use as a theme color for consistency across pages?  (This is also used here for example: https://github.com/navgurukul/bhanwari-devi/blob/432c85b158044d7dd48b260935994b5dda9188fd/src/pages/StatePartner%20Dashboard/style.scss#L128-L130)

Reply:
This will get change soon so that's why I didn't refer it.